programs/drift_vaults/src/instructions/admin_update_vault_class.rs

```diff
@@ -0,0 +1,38 @@
+use anchor_lang::prelude::*;
+
+use crate::constraints::is_admin;
+use crate::state::{Vault, VaultClass};
+use crate::{error::ErrorCode, validate};
+
+pub fn admin_update_vault_class<'info>(
```

Comment:
feel like we only want this to be set on init to avoid somebody changing it to trusted after a deposit

Reply:
permissionless to setup should be fine as long as its init only